payments-core/src/test/java/com/stripe/android/model/parsers/ConsumerPaymentDetailsJsonParserTest.kt

```diff
@@ -27,23 +27,23 @@ class ConsumerPaymentDetailsJsonParserTest {
     @Test
     fun `parse single bank account payment details`() {
         assertEquals(
-            ConsumerPaymentDetailsJsonParser
```

Comment:
Changing `expected` and `actual` parameters here for more accurate test failure outputs.